RTN-004.tex

```diff
@@ -334,19 +369,18 @@ \subsection{Policies for RSP Accounts}\label{ssec:res_pol}
 Account usage policies might include, e.g., best practices for starting and stopping servers, understanding that computational processes which overburden the system might be curtailed, abiding by storage capacity quotas, and following on- and off-boarding steps.
```

Comment:
computational processes that overburden the system

Reply:
fixed